Fix Broken Link to Service Fabric Containers samples on https://docs.microsoft.com/azure/

GitTorre opened the pull request:
The link to Service Fabric Container Samples is broken. Here is a proposed fix with a working URI that points to a doc with both Service Fabric container overview information as well as links to related samples...

azurecla commented:
Hi __@nromyn__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Jim-Parker commented:
@SnehaGunda hey Sneha, could you please review this contribution?

SnehaGunda commented:
Hi nromyn, thanks for pointing us to this, the changes you specified are correct but we don't include direct links of an Azure article in another Azure article instead we use markdown file name of that article/topic. we’re closing this pull request now, but will include your updates.

GitTorre committed:
Fix Broken Link in articles/containers/index.yml

The link to Service Fabric Container Samples is broken. Here is a proposed fix with a working URI that points to a doc with both Service Fabric container overview information as well as links to related samples...

GitTorre committed:
Merge pull request #1 from GitTorre/GitTorre-patch-1

Fix Broken Link in articles/containers/index.yml
